specs/Standards/Wallets/InjectedWallets.md

```diff
@@ -0,0 +1,325 @@
+
+> Note: Calling this method when already connected will allow users to modify their selection, triggering the 'accountsChanged' event.
+
+```ts
+const accounts = await window.near.wallet.connect();
+```
+
+##### `signTransaction`
+
+Sign a transaction. This request should require explicit approval from the user.
+
+```ts
+import { transactions, providers, utils } from "near-api-js";
+
+// Retrieve accounts (assuming already connected) and current network.
+const { network, accounts } = window.near.wallet;
+
+// Setup RPC to retrieve transaction-related prerequisites.
+const provider = new providers.JsonRpcProvider({ url: network.nodeUrl });
+
+const signedTx = await window.near.wallet.signTransaction({
+  transaction: {
+    signerId: accounts[0].accountId,
+    receiverId: "guest-book.testnet",
+    actions: [transactions.functionCall(
+      "addMessage",
+      { text: "Hello World!" },
+      utils.format.parseNearAmount("0.00000000003"),
+      utils.format.parseNearAmount("0.01")
+    )]
+  }
+});
+// Send the transaction to the blockchain.
+await provider.sendTransaction(signedTx);
+```
+
+##### `signTransactions`
+
+Sign a list of transactions. This request should require explicit approval from the user.
+
+```ts
+import { transactions, providers, utils } from "near-api-js";
+
+// Retrieve accounts (assuming already connected) and current network.
+const { network, accounts } = window.near.wallet;
+
+// Setup RPC to retrieve transaction-related prerequisites.
+const provider = new providers.JsonRpcProvider({ url: network.nodeUrl });
+
+const signedTxs = await window.near.wallet.signTransactions({
+  transactions: [
+    {
+        signerId: accounts[0].accountId,
+        receiverId: "guest-book.testnet",
+        actions: [transactions.functionCall(
+            "addMessage",
+            { text: "Hello World! (1/2)" },
+            utils.format.parseNearAmount("0.00000000003"),
+            utils.format.parseNearAmount("0.01")
+        )]
+    },
+    {
+        signerId: accounts[0].accountId,
+        receiverId: "guest-book.testnet",
+        actions: [transactions.functionCall(
+            "addMessage",
+            { text: "Hello World! (2/2)" },
+            utils.format.parseNearAmount("0.00000000003"),
+            utils.format.parseNearAmount("0.01")
+        )]
+    }
+  ]
+}); 
+
+for (let i = 0; i < signedTxs.length; i += 1) {
+  const signedTx = signedTxs[i];
+  
+  // Send the transaction to the blockchain.
+  await provider.sendTransaction(signedTx);
+}
+```
+
+##### `disconnect`
+
+Remove visibility of all accounts from the wallet.
+
+```ts
+await window.near.wallet.disconnect();
+```
+
+##### `signIn`
+
+Add `FunctionCall` access key(s) for one or more accounts. This request should require explicit approval from the user.
```

Comment:
How does the application get access to the created function call key after `signIn`?

Reply:
I was wondering the same- but after looking through the interface, the dapp key's `publicKey` is set inside the `Account` data structure which is passed to the `signIn()` method.